Core/src/Upload/AbstractUploader.php

```diff
@@ -105,7 +105,7 @@ public function __construct(
             'retries' => null,
             'requestTimeout' => null,
             'restRetryFunction' => null,
-            'retryListener' => null
+            'restRetryListener' => null
```

Comment:
```suggestion
            'retryListener' => null
```

As this option is then passed to `RequestWrappers>send()` which in turn invokes `applyHeaders()` method where this option gets consumed and unset if present.([code pointer](https://github.com/googleapis/google-cloud-php/blob/41c5bf7869d63246ec53c5523a5227828f952060/Core/src/RequestWrapper.php#L300))

Reply:
I think those are `retryHeaders` @yash30201.
Brent has mentioned `retryListerner` here.